draft-ietf-quic-transport.md

```diff
@@ -1679,9 +1683,10 @@ peer from a new local address.  In path validation, endpoints test reachability
 between a specific local address and a specific peer address, where an address
 is the two-tuple of IP address and port.
 
-Path validation tests that packets can be both sent to and received from a peer
-on the path.  Importantly, it validates that the packets received from the
-migrating endpoint do not carry a spoofed source address.
+Path validation tests that packets (PATH_CHALLENGE) can be both sent
+to and received (PATH_RESPONSE) from a peer on the path.  Importantly,
+it validates that the packets received from the migrating endpoint do
+not carry a spoofed source address.
 
```

Comment:
But that isn't exactly true since MITM can spoof the source address of these packets if it can guess the packet content (as you pointed out).

Reply:
Sure.. I didn't change this text, I just added the parentheticals to make it clearer. This PR is essentially editorial